package.json

```diff
@@ -13,12 +13,13 @@
     "mocha": "^6.2.0",
     "nyc": "^14.1.1",
     "prettier": "^1.18.2",
+    "ts-node": "^8.4.1",
     "tslint": "^5.18.0",
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^3.6.3"
   },
   "scripts": {
-    "test": "npm run tsc && nyc mocha dist/tests/*.test.js",
+    "test": "nyc mocha --require ts-node/register ./tests/*.test.ts",
     "tsc": "rm -rf dist && tsc",
     "tslint": "tslint 'src/**/*.ts?(x)'",
     "prepublishOnly": "npm run tslint && npm test"
```

Comment:
Because `npm test` no longer execute `tsc`, prepublishOnly should run it after tslint and tests are OK